demo/app.js

```diff
@@ -134,29 +134,29 @@ const Tabs = ({ tabValues, manifestUrl }) => {
   });
 
   return (
-    <section className="tabs-wrapper">
-      <div className="switcher">
+    <section className='tabs-wrapper' role='navigation' aria-label='more Ramp components in tab'>
```

Comment:
Does having the same `aria-label` here as in the child `ul` cause it to be read twice?  If so, then I could imagine it might be confusing since they have the same label.